doc/whats_new.rst

```diff
@@ -77,6 +77,12 @@ API
 
     - Add ``zorder`` parameter to :func:`mne.Evoked.plot` and derived functions to sort allow sorting channels by e.g. standard deviation, by `Jona Sassenhagen`_
 
+    - The ``baseline`` parameter of :func:`mne.Epochs.apply_baseline` is set by default (None, 0), by `Felix Raimundo`_
+
+    - Adds :func:`mne.Evoked.apply_baseline` to be consistent with :func:`mne.Epochs.apply_baseline`, by `Felix Raimundo`_
+
+    - Deprecated the `baseline` parameter in :func:`mne.Evoked.__init__`, by `Felix Raimundo`_
```

Comment:
@gamazeps you need to put the link to your homepage or github page at the bottom ortherwise the RST link will not work.